### ISSUE TITLE
Pagination start

### ISSUE THREAD

**Issue by Biromain** [ventoviro/windwalker-joomla-rad#203]
https://github.com/ventoviro/windwalker-joomla-rad/blob/staging/bundles/GeneratorBundle/Template/component/default/site/model/%7B%7Bcontroller.list.name.lower%7D%7D.php#L186

I have to fix `list.start` with

``` php
$this->set('list.start', $input->getInt('start', $input->getInt('limitstart', 0)));
```

It's a bug or a misunderstand?

**Comment by asika32764** [comment]
The contact model also use `limitstart` 
See https://github.com/joomla/joomla-cms/blob/881021635765e6b055188e6ad7d25d50a6db2cf8/components/com_contact/models/category.php#L249

But I found Joomla will delete `limitstart` in JRouterSite
https://github.com/joomla/joomla-cms/blob/881021635765e6b055188e6ad7d25d50a6db2cf8/libraries/cms/router/site.php#L635

I think it is a recently updated, I need time to try it.

**Comment by Biromain** [comment]
Ok, i think same for https://github.com/ventoviro/windwalker-joomla-rad/blob/staging/bundles/GeneratorBundle/Template/component/default/site/model/%7B%7Bcontroller.list.name.lower%7D%7D.php#L188

```php
$this->set('list.limit', $params->get('list_limit', 5) + $params->get('num_intro_articles', 0));
```

**Comment by asika32764** [comment]
Fixed in 12ee31f6ff43b35be74342341cb5fd7e9c0b3344
